### ISSUE TITLE
Am I right that this plugin doesnt work in jqmobile 1.4.5

### ISSUE THREAD

**Issue by romanmatena** [azicchetti/jquerymobile-router#91]
All events in 1.4 and up was changed, so pageshow etc. will not work. Am I right? This plugin is written only for jqmobile 1.3?

**Comment by azicchetti** [comment]
> All events in 1.4 and up was changed, so pageshow etc. will not work. Am I right? This plugin is written only for jqmobile 1.3?

Hi,
you're absolutely right.

I've stopped using jQuery Mobile since 1.3 and the plugin is more or less discontinued

A.

**Comment by romanmatena** [comment]
Thanks,

would be great to notice somewhere, bcs, it could take some hours to find what is wrong if it doesnt work.

Is there any alternative to this plugin? Or you switch jqmobile as framework? for what?

Or the solution is only to change:

line 116
                pagebeforeshow: null, pageshow: null, pagecontainershow: null,
line 232
                        selectors: "body,:jqmData(role='page'),:jqmData(role='dialog')",

and it will work? is it enough?

**Comment by azicchetti** [comment]
> Thanks,
> 
> would be great to notice somewhere, bcs, it could take some hours to find what is wrong if it doesnt work.

Yes, I'll put a notice on top, would be great to find a new maintainer and rewrite the plugin from scratch to better suit jQM >= 4.

> Is there any alternative to this plugin? Or you switch jqmobile as framework? for what?

I'm using phonejs/devexpress mobile now, life has been easier since then, eheh.
But I guess Ionic is a great framework too

> Or the solution is only to change:
> 
> I don't know, I need to find a few spare time to properly debug this issue, sorry I can't be much helpful now

Cheers,
    A.
